ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -142,19 +142,48 @@ class Metrics(val registry: MetricRegistry) {
         }
 
         object validator {
-          val prefix: MetricName = submission.prefix :+ "validator"
+          private val Prefix: MetricName = submission.prefix :+ "validator"
 
           val openEnvelope: Timer = registry.timer(prefix :+ "open_envelope")
           val acquireTransactionLock: Timer = registry.timer(prefix :+ "acquire_transaction_lock")
           val failedToAcquireTransaction: Timer =
             registry.timer(prefix :+ "failed_to_acquire_transaction")
           val releaseTransactionLock: Timer = registry.timer(prefix :+ "release_transaction_lock")
-          val validateSubmission: Timer = registry.timer(prefix :+ "validate_submission")
-          val processSubmission: Timer = registry.timer(prefix :+ "process_submission")
-          val commitSubmission: Timer = registry.timer(prefix :+ "commit_submission")
+          val validateSubmission
+            : Timer = registry.timer(prefix :+ "validate_submission") // TODO(miklos-da): Misnomer -- same as 'fetch_inputs'.
```

Comment:
@mziolekda Shall we keep these metrics separate for parallel vs. serial validation or merge them? In case of merging them I would suggest to rename `validate_submission` to `fetch_inputs`.

Reply:
I would suggest merging. I have checked and we have never made a splash about those particular metrics to anyone.

Reply:
OK, will make sure to get rid of duplicates then.

Reply:
Removed the duplicates -- @mziolekda @SamirTalwar-DA can you check if you are fine with the naming of the metrics, please?